doc/libpmemobj/pmemobj_tx_begin.3.md

```diff
@@ -81,6 +84,11 @@ TX_ONABORT
 TX_ONCOMMIT
 TX_FINALLY
 TX_END
+
+int pmemobj_tx_log_append_buffer(enum pobj_log_type type, void *addr, size_t size);
+int pmemobj_tx_log_auto_alloc(enum pobj_log_type type, int on_off);
+size_t pmemobj_tx_log_snapshots_max_size(size_t *sizes, size_t nsizes);
+size_t pmemobj_tx_log_intents_max_size(size_t nintents);
```

Comment:
I am not sure about "tx". This functions (max_size) won't be used in the transaction at all.